gqcp/include/FockSpace/SelectedFockSpace.hpp

```diff
@@ -93,14 +93,25 @@ class SelectedFockSpace : public BaseFockSpace {
     explicit SelectedFockSpace(const FrozenFockSpace& fock_space);
 
 
-    // GETTERS
+    // GETTERS & SETTERS
     size_t get_N_alpha() const { return this->N_alpha; }
     size_t get_N_beta() const { return this->N_beta; }
     const Configuration& get_configuration(size_t index) const { return this->configurations[index]; }
+    const std::vector<Configuration>& get_configurations() const { return this->configurations; }
     FockSpaceType get_type() const override { return FockSpaceType::SelectedFockSpace; }
 
+    void set_configurations(const std::vector<Configuration>& configurations) { this->configurations = configurations;
```

Comment:
I'm not really a big fan of setting the configurations this explicitly: are there use cases you really need this behavior instead of just 'adding' them?

Reply:
The current API to add is not ideal, I should just rework it instead.